cli/util/upload_bin.go

```diff
@@ -426,18 +423,16 @@ func searchObjects(ctx context.Context, p poolWrapper, containerID cid.ID, accou
 }
 
 // uploadObj uploads object to the container using provided settings.
-func uploadObj(ctx context.Context, p poolWrapper, signer user.Signer, owner util.Uint160, containerID cid.ID, objData []byte, attrs []object.Attribute) (oid.ID, error) {
+func uploadObj(ctx context.Context, p poolWrapper, signer user.Signer, ownerID user.ID, containerID cid.ID, objData []byte, attrs []object.Attribute) (oid.ID, error) {
 	var (
-		ownerID          user.ID
 		hdr              object.Object
 		prmObjectPutInit client.PrmObjectPutInit
 		resOID           = oid.ID{}
 	)
 
-	ownerID.SetScriptHash(owner)
 	hdr.SetPayload(objData)
```

Comment:
no-op

Reply:
What exactly is a no-op here?

Reply:
payload is unprocessed for `hdr`, it's written bellow. Object type system jukes now, but it will be improved soon https://github.com/nspcc-dev/neofs-sdk-go/issues/652